### PR TITLE
feature/1154 Report cu custom logo as true if enabled.

### PR DESCRIPTION
**Pull request by owenmorrill**

### CODE REVIEW

**Comment by owenmorrill** [comment]
Well, I'm missing something completely because this isn't reported anywhere locally in /statistics. Help.

**Comment by jwfuller** [comment]
@owenmorrill Check `profile_module_manager` like in https://osr-atlas03.int.colorado.edu/statistics/5898f2c7926f5b2765b2fb12

File: modules/custom/cu_atlas/cu_atlas.module [hunk 1]
```diff
@@ -176,6 +176,11 @@ function cu_atlas_atlas_statistics() {
     }
   }
 
+  // Custom Logo.
+  if (module_exists('cu_custom_logo')) {
+    $statistics_data['cu_custom_logo'] = true;
```
Comment (jwfuller) [inline modules/custom/cu_atlas/cu_atlas.module]: $statistics_data['custom_logo_settings']

**Comment by owenmorrill** [comment]
custom-logo

**Pull request merged**